lib/cosmos/interfaces/linc_interface.rb

```diff
@@ -91,16 +90,41 @@ def connect
     end
 
     def write(packet)
-      ######################################
-      # Commands handled in the COSMOS interface not the LINC target
-      ######################################
+      return if linc_interface_command(packet)
+      raise "Interface not connected" unless connected?()
 
+      # Add a GUID to the GUID field if its defined
+      # A GUID means it's an asychronous packet type.
+      if @fieldname_guid
+        guid = get_guid(packet)
+      else
+        # If @fieldname_guid is not defined (syncronous) we don't care what the
+        # GUID is because we're not trying to match it up with anything.
+        # As soon as we get a response we free the command.
+        guid = 0
```

Comment:
This else case wasn't being handled in the previous code. It worked but I wanted to call out what was happening.